poradnia/cases/tests.py

```diff
@@ -245,19 +247,34 @@ def test_record_count(self):
         self.assertTrue(hasattr(obj, 'record_count'))
         self.assertEqual(admin_obj.record_count(obj), 25)
 
-class CaseCloseTestCase(TestCase):
+
+class CaseCloseViewTestCase(TestCase):
     def setUp(self):
         self.user = UserFactory()
         self.object = CaseFactory()
         self.client.login(username=self.user.username, password='pass')
 
     def test_close_case(self):
         assign_perm('cases.can_close_case', self.user, self.object)
-        resp = self.client.post(self.object.get_close_url(), {'notify': True})
+        resp = self.client.post(self.object.get_close_url())
```

Comment:
Wygląda na to, że testy nie przechodzą bez podania parametrów w argumentach. Na pierwszy rzut oka wygląda to na bug Django (bądź konieczność większej konfiguracji w teście), ponieważ walidacja BooleanField jest traktowana inaczej niż pozostałych pól https://docs.djangoproject.com/en/1.9/ref/forms/fields/#django.forms.BooleanField